lang/src/test/java/org/kaazing/k3po/lang/internal/parser/ScriptParserImplTest.java

```diff
@@ -1242,6 +1242,29 @@ public void shouldParseConnectScriptWithComments() throws Exception {
     }
 
     @Test
+    public void shouldParseConnectScriptWithOptions() throws Exception {
+
+        String script =
+                "connect option transport \"localhost:8888\"\n" +
+                "connect tcp://localhost:7788\n" +
```

Comment:
The connect transport option should be a location, such as `tcp://localhost:8888`, right?